README.md

```diff
@@ -21,6 +21,14 @@ git clone git@github.com:lagom/online-auction-java.git
 cd online-auction-java
 sbt runAll
 ```
+## Importing into IDEs
+
+### Intellij IDEA
+You will need to install following plugins into IDEA by going to `File -> Settings -> Plugins`
+* **[SBT](http://github.com/orfjackal/idea-sbt-plugin)** : SBT build tool plugin.
```

Comment:
I think we should probably link to the documentation on the Lagom site, so that we don't have to maintain it in multiple places. I'll be merging and deploying @rstento's changes later today, and I'll let you know when that's done.